lib/matplotlib/pyplot.py

```diff
@@ -1408,6 +1408,12 @@ def axis(*v, **kwargs):
 
     The xmin, xmax, ymin, ymax tuple is returned
 
+      >>> axis('square')
```

Comment:
I have checked, the axis function is above the auto-generated part, so the changes will remain.